packages/shared/src/tracking/occurrence.ts

```diff
@@ -0,0 +1,18 @@
+import { PrimitiveValue } from '../types';
+
+
+export type OccurrenceValue =
+  | PrimitiveValue
+  | Date
+  | { [key: string]: OccurrenceValue }
+  | OccurrenceValue[];
+
+/**
+ * A container for arbitrary contextual data that can be used as a basis for dynamic evaluation
+ */
```

Comment:
This is copy-paste right? Seems to make for sense for evaluation context.

Reply:
Yep - this is still quite rough - we aren't settle on terminology yet and comments aren't updated.

Reply:
Sure this is great, just wanted to be sure!